settings/storage/address_bolt.go

```
@@ -0,0 +1,157 @@
+package storage
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/boltdb/bolt"
+)
+
+const (
+	ADDRESS_SETTING_BUCKET     string = "address_setting"
+	ADDRESS_SET_SETTING_BUCKET string = "address_set_setting"
+)
+
+type BoltAddressStorage struct {
+	db *bolt.DB
+}
+
+func NewBoltAddressStorage(dbPath string) (*BoltAddressStorage, error) {
+	var err error
+	var db *bolt.DB
+	db, err = bolt.Open(dbPath, 0600, nil)
+	if err != nil {
+		panic(err)
```

please return the error.